product_code/user_service/impl/src/main/scala/de/upb/cs/uc4/user/impl/UserSerializerRegistry.scala

```diff
@@ -3,7 +3,7 @@ package de.upb.cs.uc4.user.impl
 import com.lightbend.lagom.scaladsl.playjson.JsonSerializer
 import de.upb.cs.uc4.shared.server.SharedSerializerRegistry
 import de.upb.cs.uc4.user.impl.actor.UserState
-import de.upb.cs.uc4.user.impl.events.{ OnLatestMatriculationUpdate, OnUserCreate, OnUserDelete, OnUserUpdate }
+import de.upb.cs.uc4.user.impl.events.{ OnLatestMatriculationUpdate, OnUserCreate, OnUserForceDelete, OnUserSoftDelete, OnUserUpdate }
```

Comment:
Optimize imports to contain not more than 4 entries

```suggestion
import de.upb.cs.uc4.user.impl.events._
```